content/api/rest-api.textile

```diff
@@ -574,12 +574,107 @@ bc[json]. [{
 
 h3(#metadata-rest). Retrieve metadata for a channel
 
+h6. GET rest.ably.io/channels/@<channelId>@
+
 <%= partial partial_version('shared/_channel_metadata') %>
 
 h3(#enumeration-rest). Enumerate all active channels
 
+h6. GET rest.ably.io/channels
+
 <%= partial partial_version('shared/_channel_enumeration') %>
 
+h3(#annotations-publish). Publish annotations for a message
+
+h6. POST rest.ably.io/channels/@<channelId>@/messages/@<messageSerial>@/annotations
+
+Publish one or more annotations for a specific message identified by its serial in the @messageSerial@ url param. All annotations in a single request are published atomically and must be for the same message.
+
+See "our main annotation docs":/docs/messages/annotations for more information on annotations.
+
+The request body contains an array of annotation objects:
+
+bc[json]. [{
+  action: <number; 0 is create, 1 is delete>,
+  type: <string, specifies how this annotation should be aggregated>,
+  name: <optional string, arbitrary string that many aggregation types use to group by>,
+  count: <optional number, used with the 'multiple' aggregation type>,
+  id: <optional string, client-specified idempotency key>,
+  clientId: <optional string, explicit client identifier>,
+  data: <optional JSON-encodable data>,
+  encoding: <optional string, encoding if specifying manually>,
+  extras: <optional extras object>
+}]
+
+In JSON format, the accepted types for the @data@ payload are:
+
+* string
+* any JSON-encodable Array or Object.
+
+MessagePack additionally "supports byte arrays":https://github.com/msgpack/msgpack/blob/master/spec.md#formats-bin
+
+For annotation types and aggregation details, see "annotation types":/docs/messages/annotations#annotation-types.
+
+The message extras object is the same as for a message publish, see "message extras":/docs/api/rest-api#message-extras.
+
+The id is the same as for a message publish, see "idempotent publishing":/docs/api/rest-api#idempotent-publish
```

Comment:
Is it worth making the request body a real example and then have a table describing each property that can link out to the relevant sections noted here? I think it might make it more readable.